app/routes/articles.js

```diff
@@ -118,22 +107,30 @@ router.post('/listings', listingsLimit, (req, res) => {
         catch(err => console.log('articleListingPromise:', err))
 })
 
-router.get('/images/:id', async (req, res) => {
-    if(!req?.params?.id || !req?.user?.username) res.sendStatus(404)
-    const dirPath = path.join(__dirname + '../../content', 'articles', 'images')
-    if(!fs.existsSync(dirPath)) fs.mkdirSync(dirPath, {recursive: true})
+//GET Article Images
+router.get('/image/:id', async (req, res) => {
+    if(!req?.params?.id) return res.sendStatus(404)
+    const width = req.query?.width ?? null
```

Comment:
Isn't `req.query?.width` enough

Reply:
Since null suggests that the variable is defined, as null.
Here it's just undefined.

Reply:
Actually I believe that there will always be `req.query` so you can just do `const {width, height} = req.query`